bindgen/src/JSONBuilder.ts

```diff
@@ -151,6 +160,17 @@ export class JSONBindingsBuilder extends BaseVisitor {
     super.visit(node);
   }
 
+  make_camel_case_export(
```

Comment:
this should be camelCase

Reply:
What do you think about:

```suggestion
  camelCaseToSnakeCaseExport(
```